### ISSUE TITLE
Add an app icon for the test application

### ISSUE THREAD

**Issue by stefanieschreiner** [amosproj/amos2022ws02-automotive-test-app#61]
## User story
1. As a user
2. I want to be able to see a suitable app icon
3. So that I can recognize the test app among other apps

## Acceptance criteria
* App icon is being created and suitable as a logo for the automotive test app 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
